native/exe/SageLauncher/launcher.cpp

```diff
@@ -439,10 +456,10 @@ void doStageCleaning(BOOL bService, BOOL bClient)
 
 #define VER_SUITE_WH_SERVER 0x00008000
 
-int WINAPI WinMain( HINSTANCE hInst, 	/*Win32 entry-point routine */
-					HINSTANCE hPreInst, 
-					LPSTR lpszCmdLine, 
-					int nCmdShow )
+int WINAPI WinMain(__in HINSTANCE hInst, 	/*Win32 entry-point routine */

```

Comment:
We're not really using annotations like this anywhere that I recall, so let's not start adding them.